### DISCUSSION TITLE
Installation help

### DISCUSSION THREAD

**Discussion by quin-tessential** [winstxnhdw/lc-hax#123] [Q&A] [answered]
![image](https://github.com/winstxnhdw/lc-hax/assets/155339882/21e86aff-4dad-42ec-8a4b-5d4c9533b519)

This is what I see when I open the requirements file. I do not know what this means because I am new. Sorry to bother but may anyone help me?

![image](https://github.com/winstxnhdw/lc-hax/assets/155339882/625087ef-1a2c-4673-9086-ca43680f8ac7)

This is also what I see when I open the launch file. How do I fix this?
Thanks.

**Comment by winstxnhdw** [comment]
@quin-tessential looks like you didn't recursively clone the repository. You are also doing something extremely dangerous. You are cloning the repository in your System32 folder.. Why? That is a major security risk.

**Reply by quin-tessential** [reply]
I do not know why it cloned to System32, I do not know how to change it.

I thought I did recursively clone the repository, I put the command 

'git clone --recursive https://github.com/winstxnhdw/lc-hax.git'

into cmd. Am I doing something wrong?

**Reply by winstxnhdw** [reply]
You most likely started CMD as administrator, which defaults to the System32 path.

**Reply by quin-tessential** [reply]
Okay, I get it now. How do I change the path?

**Reply by winstxnhdw** [reply]
Just don't start CMD as administrator.

**Reply by quin-tessential** [reply]
Alright, but now I am getting these errors.

![image](https://github.com/winstxnhdw/lc-hax/assets/155339882/bea6f9ab-261f-4c96-9e3e-5e633a367a6d)

^ (requirements file)

![image](https://github.com/winstxnhdw/lc-hax/assets/155339882/cf299a63-4be4-4e6e-bfb7-fb434d9a88aa)

^ (launch file)

**Reply by winstxnhdw** [reply]
Doesn't even look like you have .NET 8.0 installed.

**Reply by quin-tessential** [reply]
![image](https://github.com/winstxnhdw/lc-hax/assets/155339882/b2dfce6e-2699-4048-8789-f5c6a71b3512)

Is it this one?

**Reply by winstxnhdw** [reply, marked as answer]
Looks correct, but it seems you have a Visual Studio configuration that's causing problems. See this: https://stackoverflow.com/questions/52376567/how-to-resolve-unable-to-find-package-nuget-error

**Reply by quin-tessential** [reply]
Thank you so much!
